Makefile

```diff
@@ -62,6 +64,14 @@ build-container-linux-%:
 		--build-arg OS=linux \
 		--build-arg ARCH=$* .
 
+# used to build cleanup controller
+build-node-cleanup-controller-linux-%:
+	CGO_ENABLED=0 GOOS=linux GOARCH=$* go build -a -ldflags '-extldflags "-static"' -mod vendor -o _output/linux/$*/local-volume-node-cleanup ./cmd/cleanup
+	$(DOCKER) buildx build --file=./cmd/cleanup/Dockerfile --platform=linux/$* \
+		-t $(STAGINGIMAGECLEANUPCONTROLLER):$(STAGINGVERSION)_linux_$* --output=type=$(OUTPUT_TYPE) \
```

Comment:
Since we're not supporting multi-arch builds initially, I would remove the "_linux" suffix from the image name.

Reply:
Done. Since I am using a distroless image https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/c40d2576273d4fa354b54f57d5f0512714c4d454/cmd/node-cleanup/Dockerfile#L15 is it even necessary to pass in the OS and arch as arguments?

Reply:
It's fine to leave it so it can be added in the future